cache/in-memory/src/event/mod.rs

```diff
@@ -41,9 +44,13 @@ impl InMemoryCache {
         let user = user.into_owned();
 
         if let Some(guild_id) = guild_id {
+            let user_id = user.id;
```

Comment:
Can this be used consistently between lines 49 and 53? Either use it in both or remove it?

Reply:
It can't be removed because the user is moved but the ID is used later. I've made this consistent.